homeassistant/components/elvia/importer.py

```diff
@@ -109,7 +109,6 @@ async def import_meter_values(self) -> None:
             if (
                 hourly_data is None
                 or len(hourly_data) == 0
-                or not hourly_data[-1]["verified"]
```

Comment:
I do not think this should be done, as this will result in wrong imports.
If these values can be overwritten when verified, that would be an option.
Alternatively, if that is not doable, for unverified, it should be its own import with a (unverified) suffix.